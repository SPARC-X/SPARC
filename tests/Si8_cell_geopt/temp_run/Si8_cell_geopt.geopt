:RELAXSTEP: 1
:CELL:   9.3272727273E+00   9.3272727273E+00   9.3272727273E+00
:VOLUME:   8.1145422690E+02
:LATVEC:
  1.0000000000E+00   0.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   1.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   0.0000000000E+00   1.0000000000E+00 
:STRESS:
 -5.7919628804E+01   4.6594186431E-07  -7.4881598009E-07 
  4.6594186431E-07  -5.7919627980E+01   2.1906349513E-06 
 -7.4881598009E-07   2.1906349513E-06  -5.7919631977E+01
:RELAXSTEP: 2
:CELL:   1.1286000000E+01   1.1286000000E+01   1.1286000000E+01
:VOLUME:   1.4375406617E+03
:LATVEC:
  1.0000000000E+00   0.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   1.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   0.0000000000E+00   1.0000000000E+00 
:STRESS:
  1.1182582876E+01   8.1610301464E-11   8.6431779807E-11 
  8.1610301464E-11   1.1182582876E+01   8.1615640337E-11 
  8.6431779807E-11   8.1615640337E-11   1.1182582876E+01
:RELAXSTEP: 3
:CELL:   1.1014370093E+01   1.1014370093E+01   1.1014370093E+01
:VOLUME:   1.3362231612E+03
:LATVEC:
  1.0000000000E+00   0.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   1.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   0.0000000000E+00   1.0000000000E+00 
:STRESS:
  8.3164476078E+00   2.9670654716E-12   8.7014469268E-13 
  2.9670654716E-12   8.3164476078E+00   3.5082850505E-12 
  8.7014469268E-13   3.5082850505E-12   8.3164476078E+00
:RELAXSTEP: 4
:CELL:   1.0257163206E+01   1.0257163206E+01   1.0257163206E+01
:VOLUME:   1.0791499563E+03
:LATVEC:
  1.0000000000E+00   0.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   1.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   0.0000000000E+00   1.0000000000E+00 
:STRESS:
 -8.3473462075E+00   2.4922596799E-12  -2.2318533737E-12 
  2.4922596799E-12  -8.3473462075E+00   2.3200302465E-12 
 -2.2318533737E-12   2.3200302465E-12  -8.3473462075E+00
:RELAXSTEP: 5
:CELL:   1.0649927324E+01   1.0649927324E+01   1.0649927324E+01
:VOLUME:   1.2079248959E+03
:LATVEC:
  1.0000000000E+00   0.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   1.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   0.0000000000E+00   1.0000000000E+00 
:STRESS:
  2.2386457589E+00  -5.6373715402E-12   8.5629682192E-12 
 -5.6373715402E-12   2.2386457589E+00   6.3353809746E-12 
  8.5629682192E-12   6.3353809746E-12   2.2386457589E+00
:RELAXSTEP: 6
:CELL:   1.0539269458E+01   1.0539269458E+01   1.0539269458E+01
:VOLUME:   1.1706620096E+03
:LATVEC:
  1.0000000000E+00   0.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   1.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   0.0000000000E+00   1.0000000000E+00 
:STRESS:
 -2.5301736720E-01  -6.1741549746E-12  -1.1222023499E-12 
 -6.1741549746E-12  -2.5301736722E-01  -3.8086483177E-12 
 -1.1222023499E-12  -3.8086483177E-12  -2.5301736727E-01
:RELAXSTEP: 7
:CELL:   1.0550612457E+01   1.0550612457E+01   1.0550612457E+01
:VOLUME:   1.1744458909E+03
:LATVEC:
  1.0000000000E+00   0.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   1.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   0.0000000000E+00   1.0000000000E+00 
:STRESS:
  1.8538948800E-02  -3.2897833427E-12  -4.5014600386E-12 
 -3.2897833427E-12   1.8538948772E-02   2.3669756821E-12 
 -4.5014600386E-12   2.3669756821E-12   1.8538948777E-02
:RELAXSTEP: 8
:CELL:   1.0549838854E+01   1.0549838854E+01   1.0549838854E+01
:VOLUME:   1.1741875681E+03
:LATVEC:
  1.0000000000E+00   0.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   1.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   0.0000000000E+00   1.0000000000E+00 
:STRESS:
  1.9084729640E-04  -2.1432367476E-12  -3.4614516441E-13 
 -2.1432367476E-12   1.9084726654E-04   4.4695321541E-12 
 -3.4614516441E-13   4.4695321541E-12   1.9084722239E-04
:RELAXSTEP: 9
:CELL:   1.0549830813E+01   1.0549830813E+01   1.0549830813E+01
:VOLUME:   1.1741848832E+03
:LATVEC:
  1.0000000000E+00   0.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   1.0000000000E+00   0.0000000000E+00 
  0.0000000000E+00   0.0000000000E+00   1.0000000000E+00 
:STRESS:
 -6.0375280744E-05  -1.8794664891E-12  -1.6536134380E-12 
 -1.8794664891E-12  -6.0375355207E-05   2.3418735055E-12 
 -1.6536134380E-12   2.3418735055E-12  -6.0375296872E-05
