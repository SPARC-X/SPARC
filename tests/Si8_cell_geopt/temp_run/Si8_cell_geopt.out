***************************************************************************
*                       SPARC (version Apr 23, 2021)                      *
*   Copyright (c) 2020 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                   Start time: Mon May 10 20:43:23 2021                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 10.26 10.26 10.26 
LATVEC:
1 0 0 
0 1 0 
0 0 1 
FD_GRID: 103 103 103
FD_ORDER: 12
BC: P P P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
SPIN_TYP: 0
ELEC_TEMP_TYPE: Fermi-Dirac
SMEARING: 0.001000000336
CHEB_DEGREE: 50
NSTATES: 24
EXCHANGE_CORRELATION: GGA_PBE
RELAX_FLAG: 2
CALC_STRESS: 1
CHEFSI_BOUND_FLAG: 0
MAXIT_SCF: 100
MINIT_SCF: 3
MAXIT_POISSON: 3000
TOL_SCF: 1.00E-06
POISSON_SOLVER: AAR
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: density
MIXING_PRECOND: kerker
TOL_PRECOND: 9.92E-06
PRECOND_KERKER_KTF: 1
PRECOND_KERKER_THRESH: 0.1
MIXING_PARAMETER: 0.3
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART: 0
REFERENCE_CUTOFF: 0.5
RHO_TRIGGER: 4
FIX_RAND: 0
VERBOSITY: 1
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
PRINT_VELS: 1
PRINT_RESTART: 0
TOL_RELAX_CELL: 1.00E-04
RELAX_MAXDILAT: 1.10E+00
PRINT_RELAXOUT: 1
OUTPUT_FILE: Si8_cell_geopt
***************************************************************************
                           Parallelization                                 
***************************************************************************
NP_SPIN_PARAL: 1
NP_KPOINT_PARAL: 1
NP_BAND_PARAL: 24
NP_DOMAIN_PARAL: 1 1 1
NP_DOMAIN_PHI_PARAL: 2 3 4
EIG_SERIAL_MAXNS: 2000
***************************************************************************
                             Initialization                                
***************************************************************************
Number of processors               :  24
Mesh spacing                       :  0.0996117 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  Si8_cell_geopt.out
Total number of atom types         :  1
Total number of atoms              :  8
Total number of electrons          :  32
Atom type 1  (valence electrons)   :  Si 4
Pseudopotential                    :  Si.psp8
Pseudocharge radii of atom type 1  :  6.38 6.38 6.38 (x, y, z dir)
Number of atoms of type 1          :  8
Estimated total memory usage       :  1.40 GB
Estimated memory per processor     :  59.75 MB
***************************************************************************
                         Reinitialized parameters                          
***************************************************************************
CELL: 9.32727272727273 9.32727272727273 9.32727272727273 
CHEB_DEGREE: 50
***************************************************************************
                             Reinitialization                              
***************************************************************************
Mesh spacing                       :  0.090556 (Bohr)
===================================================================
                    Self Consistent Field (SCF#1)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -3.7403711635E+00        2.271E-01        18.199
2            -3.8647222850E+00        1.972E-01        5.682
3            -3.8892153670E+00        1.700E-01        5.745
4            -3.8923872033E+00        1.263E-01        5.721
5            -3.8929434169E+00        7.938E-02        5.683
6            -3.8931262958E+00        5.940E-02        5.637
7            -3.8931379488E+00        3.951E-02        5.633
8            -3.8932070779E+00        2.740E-02        5.623
9            -3.8933143341E+00        2.622E-02        5.583
10           -3.8933896383E+00        1.880E-02        5.579
11           -3.8934557374E+00        1.559E-02        5.568
12           -3.8934962406E+00        7.222E-03        5.543
13           -3.8935215863E+00        3.119E-03        5.419
14           -3.8935359475E+00        2.975E-03        5.386
15           -3.8935446998E+00        2.317E-03        5.402
16           -3.8935497656E+00        2.401E-03        5.484
17           -3.8935529916E+00        2.097E-03        5.439
18           -3.8935543281E+00        1.388E-03        5.408
19           -3.8935552557E+00        6.413E-04        5.353
20           -3.8935558275E+00        5.543E-04        5.345
21           -3.8935561754E+00        5.144E-04        5.264
22           -3.8935563899E+00        5.211E-04        5.225
23           -3.8935564592E+00        5.477E-04        5.141
24           -3.8935565226E+00        5.839E-04        5.276
25           -3.8935564872E+00        5.397E-04        5.314
26           -3.8935563783E+00        4.590E-04        5.283
27           -3.8935563243E+00        4.018E-04        5.310
28           -3.8935561990E+00        2.908E-04        5.337
29           -3.8935561296E+00        2.212E-04        5.339
30           -3.8935560996E+00        1.319E-04        5.244
31           -3.8935561025E+00        9.409E-05        5.267
32           -3.8935561000E+00        6.069E-05        5.230
33           -3.8935560986E+00        4.609E-05        5.173
34           -3.8935561157E+00        3.066E-05        5.171
35           -3.8935561163E+00        2.377E-05        5.174
36           -3.8935561049E+00        1.282E-05        5.114
37           -3.8935560997E+00        1.014E-05        5.081
38           -3.8935560974E+00        8.850E-06        5.007
39           -3.8935560988E+00        7.422E-06        5.043
40           -3.8935561001E+00        6.198E-06        5.047
41           -3.8935561010E+00        3.502E-06        4.886
42           -3.8935561023E+00        3.022E-06        4.935
43           -3.8935560989E+00        2.730E-06        4.902
44           -3.8935560973E+00        2.455E-06        4.894
45           -3.8935560964E+00        1.946E-06        4.917
46           -3.8935561001E+00        1.692E-06        5.012
47           -3.8935560996E+00        3.518E-06        4.836
48           -3.8935561030E+00        2.341E-06        4.842
49           -3.8935560950E+00        1.292E-06        4.839
50           -3.8935560958E+00        6.686E-07        4.129
Total number of SCF: 50    

k = [0.000, 0.000, 0.000]
Occupation of state 22 = 0.211388449183886.
Occupation of state 24 = 0.000000000000000.
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -3.8935560936E+00 (Ha/atom)
Total free energy                  : -3.1148448749E+01 (Ha)
Band structure energy              :  4.2196140881E+00 (Ha)
Exchange correlation energy        : -1.0374673574E+01 (Ha)
Self and correction energy         : -1.0219616022E+02 (Ha)
Entropy*kb*T                       : -7.1444217807E-03 (Ha)
Fermi level                        :  3.4167214315E-01 (Ha)
RMS force                          :  2.6417216613E-07 (Ha/Bohr)
Maximum force                      :  3.0892668492E-07 (Ha/Bohr)
Time for force calculation         :  0.225 (sec)
Pressure                           :  5.7919629587E+01 (GPa)
Maximum stress                     :  5.7919631977E+01 (GPa)
Time for stress calculation        :  0.560 (sec)
***************************************************************************
                         Reinitialized parameters                          
***************************************************************************
CELL: 11.286 11.286 11.286 
CHEB_DEGREE: 50
***************************************************************************
                             Reinitialization                              
***************************************************************************
Mesh spacing                       :  0.109573 (Bohr)
===================================================================
                    Self Consistent Field (SCF#2)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -3.9444819910E+00        3.064E-01        4.121
2            -3.9369495361E+00        2.276E-01        3.970
3            -3.9213692430E+00        5.441E-02        3.917
4            -3.9207632077E+00        3.156E-02        3.856
5            -3.9215998979E+00        9.481E-03        3.682
6            -3.9216116176E+00        6.697E-03        3.746
7            -3.9216240959E+00        7.125E-03        3.729
8            -3.9216306083E+00        1.208E-03        3.612
9            -3.9216307946E+00        1.189E-03        3.650
10           -3.9216307578E+00        1.779E-04        3.593
11           -3.9216307272E+00        2.006E-04        3.612
12           -3.9216307162E+00        3.445E-05        3.547
13           -3.9216307739E+00        1.316E-05        3.492
14           -3.9216307527E+00        2.254E-06        3.485
15           -3.9216307598E+00        7.171E-07        2.898
Total number of SCF: 15    

k = [0.000, 0.000, 0.000]
Occupation of state 22 = 0.000000001085750.
Occupation of state 24 = 0.000000000000000.
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -3.9216307650E+00 (Ha/atom)
Total free energy                  : -3.1373046120E+01 (Ha)
Band structure energy              : -7.2054007290E-01 (Ha)
Exchange correlation energy        : -9.2477934357E+00 (Ha)
Self and correction energy         : -1.0219599148E+02 (Ha)
Entropy*kb*T                       : -3.0667813101E-10 (Ha)
Fermi level                        :  1.4972056892E-01 (Ha)
RMS force                          :  2.3226339274E-07 (Ha/Bohr)
Maximum force                      :  4.6143704501E-07 (Ha/Bohr)
Time for force calculation         :  0.147 (sec)
Pressure                           : -1.1182582876E+01 (GPa)
Maximum stress                     :  1.1182582876E+01 (GPa)
Time for stress calculation        :  0.367 (sec)
***************************************************************************
                         Reinitialized parameters                          
***************************************************************************
CELL: 11.0143700929061 11.0143700929061 11.0143700929061 
CHEB_DEGREE: 50
***************************************************************************
                             Reinitialization                              
***************************************************************************
Mesh spacing                       :  0.106936 (Bohr)
===================================================================
                    Self Consistent Field (SCF#3)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -3.9261436562E+00        5.104E-02        4.032
2            -3.9260797462E+00        2.995E-02        3.954
3            -3.9259284087E+00        1.527E-02        3.922
4            -3.9258754993E+00        3.734E-03        3.857
5            -3.9258785890E+00        4.058E-03        3.915
6            -3.9258753664E+00        1.320E-03        3.766
7            -3.9258752616E+00        3.061E-04        3.710
8            -3.9258753457E+00        2.529E-05        3.635
9            -3.9258753580E+00        3.802E-05        3.600
10           -3.9258753332E+00        6.308E-06        3.549
11           -3.9258753041E+00        1.497E-05        3.503
12           -3.9258753450E+00        8.443E-06        3.503
13           -3.9258753461E+00        4.552E-06        3.464
14           -3.9258753251E+00        3.372E-06        3.496
15           -3.9258753434E+00        2.927E-06        3.557
16           -3.9258753592E+00        2.682E-06        3.537
17           -3.9258753059E+00        1.172E-06        3.449
18           -3.9258753793E+00        8.991E-07        3.000
Total number of SCF: 18    

k = [0.000, 0.000, 0.000]
Occupation of state 22 = 0.000000004051047.
Occupation of state 24 = 0.000000000000000.
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -3.9258753364E+00 (Ha/atom)
Total free energy                  : -3.1407002691E+01 (Ha)
Band structure energy              : -1.9055324075E-01 (Ha)
Exchange correlation energy        : -9.3860933596E+00 (Ha)
Self and correction energy         : -1.0219600160E+02 (Ha)
Entropy*kb*T                       : -1.0371396818E-09 (Ha)
Fermi level                        :  1.7073299607E-01 (Ha)
RMS force                          :  2.0863388178E-07 (Ha/Bohr)
Maximum force                      :  4.1726485884E-07 (Ha/Bohr)
Time for force calculation         :  0.155 (sec)
Pressure                           : -8.3164476078E+00 (GPa)
Maximum stress                     :  8.3164476078E+00 (GPa)
Time for stress calculation        :  0.385 (sec)
***************************************************************************
                         Reinitialized parameters                          
***************************************************************************
CELL: 10.2571632062197 10.2571632062197 10.2571632062197 
CHEB_DEGREE: 50
***************************************************************************
                             Reinitialization                              
***************************************************************************
Mesh spacing                       :  0.0995841 (Bohr)
===================================================================
                    Self Consistent Field (SCF#4)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -3.9290224078E+00        1.454E-01        4.519
2            -3.9289788827E+00        7.916E-02        4.365
3            -3.9279453598E+00        4.556E-02        4.363
4            -3.9274754446E+00        9.376E-03        4.292
5            -3.9275444116E+00        1.540E-02        4.306
6            -3.9274961820E+00        3.226E-03        4.250
7            -3.9274972558E+00        1.010E-03        4.148
8            -3.9274975927E+00        7.802E-05        4.076
9            -3.9274976584E+00        1.206E-04        4.018
10           -3.9274976980E+00        2.472E-05        3.993
11           -3.9274976737E+00        3.345E-05        3.923
12           -3.9274976561E+00        1.859E-05        3.951
13           -3.9274976851E+00        9.763E-06        3.915
14           -3.9274976772E+00        4.182E-06        3.860
15           -3.9274976245E+00        5.226E-06        3.902
16           -3.9274976689E+00        1.130E-06        3.818
17           -3.9274976032E+00        1.383E-06        3.836
18           -3.9274976422E+00        4.529E-07        3.362
Total number of SCF: 18    

k = [0.000, 0.000, 0.000]
Occupation of state 22 = 0.000001713459557.
Occupation of state 24 = 0.000000000000000.
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -3.9274976398E+00 (Ha/atom)
Total free energy                  : -3.1419981118E+01 (Ha)
Band structure energy              :  1.5501609065E+00 (Ha)
Exchange correlation energy        : -9.8227055318E+00 (Ha)
Self and correction energy         : -1.0219604549E+02 (Ha)
Entropy*kb*T                       : -3.0028768060E-07 (Ha)
Fermi level                        :  2.3861918779E-01 (Ha)
RMS force                          :  1.5473620887E-07 (Ha/Bohr)
Maximum force                      :  3.0944627315E-07 (Ha/Bohr)
Time for force calculation         :  0.177 (sec)
Pressure                           :  8.3473462075E+00 (GPa)
Maximum stress                     :  8.3473462075E+00 (GPa)
Time for stress calculation        :  0.437 (sec)
***************************************************************************
                         Reinitialized parameters                          
***************************************************************************
CELL: 10.6499273239233 10.6499273239233 10.6499273239233 
CHEB_DEGREE: 50
***************************************************************************
                             Reinitialization                              
***************************************************************************
Mesh spacing                       :  0.103397 (Bohr)
===================================================================
                    Self Consistent Field (SCF#5)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -3.9295565966E+00        6.727E-02        4.247
2            -3.9293469063E+00        4.203E-02        4.140
3            -3.9289803686E+00        1.886E-02        4.092
4            -3.9288878324E+00        5.314E-03        4.073
5            -3.9289054812E+00        2.153E-03        4.065
6            -3.9289052924E+00        1.835E-03        3.971
7            -3.9289050628E+00        1.447E-04        3.853
8            -3.9289050409E+00        1.697E-04        3.845
9            -3.9289050763E+00        4.066E-05        3.774
10           -3.9289050273E+00        1.391E-05        3.735
11           -3.9289050877E+00        6.031E-06        3.660
12           -3.9289050294E+00        2.386E-06        3.667
13           -3.9289050428E+00        2.132E-06        3.549
14           -3.9289050810E+00        1.655E-06        3.631
15           -3.9289050459E+00        1.285E-06        3.606
16           -3.9289050316E+00        1.008E-06        3.595
17           -3.9289050547E+00        7.749E-07        3.160
Total number of SCF: 17    

k = [0.000, 0.000, 0.000]
Occupation of state 22 = 0.000000046934478.
Occupation of state 24 = 0.000000000000000.
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -3.9289050559E+00 (Ha/atom)
Total free energy                  : -3.1431240447E+01 (Ha)
Band structure energy              :  5.9443509910E-01 (Ha)
Exchange correlation energy        : -9.5865722842E+00 (Ha)
Self and correction energy         : -1.0219601899E+02 (Ha)
Entropy*kb*T                       : -1.0285809642E-08 (Ha)
Fermi level                        :  2.0153689511E-01 (Ha)
RMS force                          :  1.8232664576E-07 (Ha/Bohr)
Maximum force                      :  3.6465572202E-07 (Ha/Bohr)
Time for force calculation         :  0.163 (sec)
Pressure                           : -2.2386457589E+00 (GPa)
Maximum stress                     :  2.2386457589E+00 (GPa)
Time for stress calculation        :  0.408 (sec)
***************************************************************************
                         Reinitialized parameters                          
***************************************************************************
CELL: 10.539269458009 10.539269458009 10.539269458009 
CHEB_DEGREE: 50
***************************************************************************
                             Reinitialization                              
***************************************************************************
Mesh spacing                       :  0.102323 (Bohr)
===================================================================
                    Self Consistent Field (SCF#6)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -3.9291132579E+00        1.994E-02        4.250
2            -3.9291011713E+00        1.177E-02        4.192
3            -3.9290753258E+00        5.940E-03        4.143
4            -3.9290669476E+00        1.502E-03        4.066
5            -3.9290678215E+00        1.300E-03        4.069
6            -3.9290674799E+00        4.664E-04        3.980
7            -3.9290674191E+00        1.266E-04        3.922
8            -3.9290674842E+00        2.443E-05        3.793
9            -3.9290674563E+00        1.635E-05        3.817
10           -3.9290674744E+00        2.402E-06        3.754
11           -3.9290674724E+00        5.075E-06        3.739
12           -3.9290674756E+00        2.567E-06        3.731
13           -3.9290674605E+00        1.522E-06        3.653
14           -3.9290675110E+00        1.110E-06        3.727
15           -3.9290674824E+00        2.402E-07        3.211
Total number of SCF: 15    

k = [0.000, 0.000, 0.000]
Occupation of state 22 = 0.000000116262359.
Occupation of state 24 = 0.000000000000000.
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -3.9290674360E+00 (Ha/atom)
Total free energy                  : -3.1432539488E+01 (Ha)
Band structure energy              :  8.5138833937E-01 (Ha)
Exchange correlation energy        : -9.6509460410E+00 (Ha)
Self and correction energy         : -1.0219602548E+02 (Ha)
Entropy*kb*T                       : -2.4160875698E-08 (Ha)
Fermi level                        :  2.1154575066E-01 (Ha)
RMS force                          :  1.7599303054E-07 (Ha/Bohr)
Maximum force                      :  3.5198541665E-07 (Ha/Bohr)
Time for force calculation         :  0.168 (sec)
Pressure                           :  2.5301736723E-01 (GPa)
Maximum stress                     :  2.5301736727E-01 (GPa)
Time for stress calculation        :  0.423 (sec)
***************************************************************************
                         Reinitialized parameters                          
***************************************************************************
CELL: 10.5506124570726 10.5506124570726 10.5506124570726 
CHEB_DEGREE: 50
***************************************************************************
                             Reinitialization                              
***************************************************************************
Mesh spacing                       :  0.102433 (Bohr)
===================================================================
                    Self Consistent Field (SCF#7)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -3.9290697942E+00        2.017E-03        4.221
2            -3.9290696583E+00        1.207E-03        4.082
3            -3.9290693682E+00        5.935E-04        4.012
4            -3.9290692849E+00        1.545E-04        3.977
5            -3.9290693369E+00        1.121E-04        3.935
6            -3.9290693196E+00        4.840E-05        3.855
7            -3.9290693857E+00        9.841E-06        3.789
8            -3.9290692875E+00        3.297E-06        3.731
9            -3.9290693635E+00        1.456E-06        3.715
10           -3.9290693276E+00        2.139E-07        3.216
Total number of SCF: 10    

k = [0.000, 0.000, 0.000]
Occupation of state 22 = 0.000000105608716.
Occupation of state 24 = 0.000000000000000.
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -3.9290693776E+00 (Ha/atom)
Total free energy                  : -3.1432555021E+01 (Ha)
Band structure energy              :  8.2462417054E-01 (Ha)
Exchange correlation energy        : -9.6442703122E+00 (Ha)
Self and correction energy         : -1.0219602478E+02 (Ha)
Entropy*kb*T                       : -2.2059701216E-08 (Ha)
Fermi level                        :  2.1050529252E-01 (Ha)
RMS force                          :  1.7973811174E-07 (Ha/Bohr)
Maximum force                      :  3.5945986821E-07 (Ha/Bohr)
Time for force calculation         :  0.163 (sec)
Pressure                           : -1.8538948783E-02 (GPa)
Maximum stress                     :  1.8538948800E-02 (GPa)
Time for stress calculation        :  0.416 (sec)
***************************************************************************
                         Reinitialized parameters                          
***************************************************************************
CELL: 10.5498388544354 10.5498388544354 10.5498388544354 
CHEB_DEGREE: 50
***************************************************************************
                             Reinitialization                              
***************************************************************************
Mesh spacing                       :  0.102426 (Bohr)
===================================================================
                    Self Consistent Field (SCF#8)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -3.9290694050E+00        1.375E-04        4.096
2            -3.9290693779E+00        8.227E-05        3.927
3            -3.9290693820E+00        4.048E-05        3.895
4            -3.9290693840E+00        1.054E-05        3.865
5            -3.9290693882E+00        7.683E-06        3.833
6            -3.9290693904E+00        3.259E-06        3.774
7            -3.9290693393E+00        6.859E-07        3.214
Total number of SCF: 7     

k = [0.000, 0.000, 0.000]
Occupation of state 22 = 0.000000106314445.
Occupation of state 24 = 0.000000000000000.
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -3.9290693541E+00 (Ha/atom)
Total free energy                  : -3.1432554833E+01 (Ha)
Band structure energy              :  8.2645024297E-01 (Ha)
Exchange correlation energy        : -9.6447245843E+00 (Ha)
Self and correction energy         : -1.0219602483E+02 (Ha)
Entropy*kb*T                       : -2.2197086667E-08 (Ha)
Fermi level                        :  2.1057635677E-01 (Ha)
RMS force                          :  1.7964427509E-07 (Ha/Bohr)
Maximum force                      :  3.5928977402E-07 (Ha/Bohr)
Time for force calculation         :  0.163 (sec)
Pressure                           : -1.9084726178E-04 (GPa)
Maximum stress                     :  1.9084729640E-04 (GPa)
Time for stress calculation        :  0.418 (sec)
***************************************************************************
                         Reinitialized parameters                          
***************************************************************************
CELL: 10.5498308132985 10.5498308132985 10.5498308132985 
CHEB_DEGREE: 50
***************************************************************************
                             Reinitialization                              
***************************************************************************
Mesh spacing                       :  0.102426 (Bohr)
===================================================================
                    Self Consistent Field (SCF#9)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -3.9290693745E+00        1.061E-06        3.914
2            -3.9290693732E+00        6.955E-07        3.744
3            -3.9290693609E+00        2.610E-07        3.217
Total number of SCF: 3     

k = [0.000, 0.000, 0.000]
Occupation of state 22 = 0.000000106323998.
Occupation of state 24 = 0.000000000000000.
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -3.9290693520E+00 (Ha/atom)
Total free energy                  : -3.1432554816E+01 (Ha)
Band structure energy              :  8.2646753489E-01 (Ha)
Exchange correlation energy        : -9.6447298623E+00 (Ha)
Self and correction energy         : -1.0219602483E+02 (Ha)
Entropy*kb*T                       : -2.2198507684E-08 (Ha)
Fermi level                        :  2.1057705895E-01 (Ha)
RMS force                          :  1.7967259986E-07 (Ha/Bohr)
Maximum force                      :  3.5934531331E-07 (Ha/Bohr)
Time for force calculation         :  0.163 (sec)
Pressure                           :  6.0375310941E-05 (GPa)
Maximum stress                     :  6.0375355207E-05 (GPa)
Time for stress calculation        :  0.418 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  682.166 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Collaborators: J.E. Pask (LLNL), A.J. Medford (GT), E. Chow (GT)    *
*  Acknowledgements: U.S. DOE (DE-SC0019410); U.S. NSF (1333500,1553212)  *
***************************************************************************
                                                                           
